### PR TITLE
Replace transulcent colors with equivalent solid colors 

### PR DESCRIPTION
**Pull request by Pante**
**Describe the changes**
This addresses the following:
* Replace translucent colors with equivalent solid colors - using translucent colors may result in unexpected behavior when using the affected widgets against certain backgrounds.
* Introduce methods for getting equivalent solid colors of translucent colors.
* Fix inconsistent naming across widget styles.
* Refine blue screen tests.

**Checklist**
- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md).
- [x] I have included the relevant unit/golden tests.
- [x] I have included the relevant samples.
- [x] I have updated the documentation accordingly.
- [x] I have updated the [CHANGELOG.md](../forui/CHANGELOG.md) if necessary.

### CODE REVIEW

File: forui/lib/src/theme/color_scheme.dart [hunk 1]
```diff
@@ -27,9 +27,6 @@ final class FColorScheme with Diagnosticable {
   /// This is typically used to determine the appearance of native UI elements such as on-screen keyboards.
   final Brightness brightness;
 
-  /// The percentage, between `0` and `1`, used to set a color's lightness to derive a disabled color.
-  final double disabledColorLightness;
```
Comment (kawaijoe) [inline forui/lib/src/theme/color_scheme.dart:31]: Do you think it's still nice to have this? We could have another for the hover state as well.

File: samples/lib/widgets/select_group.dart [hunk 2]
```diff
@@ -73,7 +73,7 @@ class CheckboxFormState extends State<CheckboxForm> {
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            FSelectGroup(
+            FSelectGroup<Language>(
```
Comment (kawaijoe) [inline samples/lib/widgets/select_group.dart:76]: Do we need to type this? Sad if we do :(

**Review by kawaijoe — COMMENTED** [review]
Generally looks good, just a few thoughts. If we do decide to include the fields to adjust the disabled and hover opacity, I think we should include a short section under the theme page of the docs.

File: forui/lib/src/theme/color_scheme.dart (hunk 1 above)
Comment (Pante) [reply forui/lib/src/theme/color_scheme.dart:31]: Ah, to make the opacity configurable?

File: samples/lib/widgets/select_group.dart (hunk 2 above)
Comment (Pante) [reply samples/lib/widgets/select_group.dart:76]: Yes sadly :( Dart's type inference fails here.

File: forui/lib/src/theme/color_scheme.dart (hunk 1 above)
Comment (kawaijoe) [reply forui/lib/src/theme/color_scheme.dart:31]: Please add the rename to the `CHANGELOG`

File: samples/lib/widgets/select_group.dart (hunk 2 above)
Comment (kawaijoe) [reply samples/lib/widgets/select_group.dart:76]: :o I don't remember it throwing any compile time errors. Could you update the code sample in the docs, I don't think I have explicit generics included.

File: docs/pages/docs/themes.mdx [hunk 3]
```diff
@@ -201,6 +201,14 @@ Widget build(BuildContext context) {
...
 ```
 
+#### Hovered and Disabled Colors
+
+Hovered and disabled colors are derived by adjusting the opacity. To derive these colors, use the `FColorScheme.hover`
+and `FColorScheme.disable` methods. The opacity can be adjusted with `FColorScheme.enabledHoveredOpacity` and
+`FColorScheme.disabledOpacity`.
+
+```dart filename="main.dart" {4-6}
+
```
Comment (kawaijoe) [inline docs/pages/docs/themes.mdx:211]: Might be a mistake here.

**Review by Daviiddoo — APPROVED** [review]

**Pull request merged**